test/README.md

```diff
@@ -0,0 +1,144 @@
+
+**********This file describes the work done and steps to perform test on the robot localization package.**********
```

Comment:
*for ROS2

Reply:
Done in https://github.com/cra-ros-pkg/robot_localization/commit/87f41f0655e31fb247f06dc24f31a4d3fc7af83b